libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -36,7 +36,7 @@ const AP_Param::GroupInfo HarmonicNotchFilterParams::var_info[] = {
 
     // @Param: FREQ
     // @DisplayName: Harmonic Notch Filter base frequency
-    // @Description: Harmonic Notch Filter base center frequency in Hz. This should be set at most half the backend gyro rate (which is typically 1Khz). For helicopters using RPM sensor to dynamically set the notch frequency, use this parameter to provide a lower limit to the dynamic notch filter.  Recommend setting it to half the operating rotor speed in Hz.
+    // @Description: Harmonic Notch Filter base center frequency in Hz. This should be set at most half the backend gyro rate (which is typically 1Khz). When using RPM sensor tracking MODE this provides a lower limit to for frequency tracking, helicopters using RPM MODE should set this to half the operating rotor speed.
```

Comment:
a little better perhaps?
```suggestion
    // @Description: Harmonic Notch Filter base center frequency in Hz. This should be set at most half the backend gyro rate (which is typically 1Khz). When using RPM sensor tracking MODE this provides a lower limit to for frequency tracking. Traditional helicopters using RPM MODE should set this to half the operating rotor speed.
```

Reply:
I can see this getting confused with ESC RPMs. Maybe you should say ```RPM sensor (usually only used on traditional helicopters)``` or something?

Reply:
Isn't it the same for ESC RPM though? I.e. it also sets the minimum frequency?

Reply:
@IamPete1 @andyp1per what do you set that center freq to for dynamic notch on multi’s?  As @MichelleRos says, it should still be the min freq. 

@Hwurzburg why do you add “Traditional”. The set up is true for any helicopter whether it is traditional (single) or dual heli. I think leaving it as just helicopter is fine.

Reply:
@MichelleRos Your right, looks like it provides minimum for rpm, ESC and fft.  

I'm quite tempted to say we should just remove the last two sentences as confusing and have a more complete explanation on the wiki.

Reply:
Yes sets minimum for everything